src/beqs.cpp

```diff
@@ -107,6 +109,20 @@ Rcpp::DataFrame processResponseEvent(Event event, const bool verbose) {
                     chk[i] = true;
                 }
             }
+            all_chk = all_chk and chk[i];
+        }
+
+        if(all_chk){
+            break;
+        }
+    }
+
+    if(!all_chk){               // Check if any columns have not been checked successfully - these will all set to fallback NA
+        for (int i=0; i<cols; i++) {
+            if(!chk[i]){
+                lst[i] = Rcpp::CharacterVector(rows, R_NaString);
```

Comment:
That doesn't work. 

We may be one of four types here. You are assuming it is always character when it might be numeric or date or ... and you cannot assign a character-NA to these others.

We might be able to assign `R_NilValue`.